src/main/java/com/salesforce/dynamodbv2/mt/mappers/CreateTableRequestBuilder.java

```diff
@@ -221,7 +221,7 @@ private void addAttributeDefinition(String field, ScalarAttributeType fieldType)
         addAttributeDefinition(primaryKey.getHashKey(), primaryKey.getHashKeyType());
         if (primaryKey.getRangeKey().isPresent()) {
             keySchemaElements.add(new KeySchemaElement(primaryKey.getRangeKey().get(), RANGE));
-            addAttributeDefinition(primaryKey.getRangeKey().get(), primaryKey.getRangeKeyType().get());
+            addAttributeDefinition(primaryKey.getRangeKey().get(), primaryKey.getRangeKeyType().orElseThrow());
```

Comment:
Taking advantage of "Since jdk10" to address code style violation of "Optional get() without isPresent()".